### DISCUSSION TITLE
Feature Request: Forwarding Sponsorships

### DISCUSSION THREAD

**Discussion by patak-cat** [community/community#10177] [Other Feature Feedback, Questions, & Ideas]
It would be good if we could forward sponsorships. Some maintainers may want to redistribute to others some of the donations they are receiving, and fees shouldn't apply when sponsoring someone through forwarding (and neither the need to get the money out of Github and re-send it in another transaction). The UI could hint you about close devs, and make this flow easily discoverable so more maintainers will use it.

IMO it could help to surface key maintainers that don't have the visibility they deserve.

**Comment by FagnerMartinsBrack** [comment]
A real use case https://github.com/js-cookie/js-cookie/discussions/755

**Comment by jlord** [comment]
This is a feedback and something we're starting to think through 🎉  Cross-linking to https://github.com/github/feedback/discussions/12217.

**Reply by jd-solanki** [reply]
Hi,

I am developing open source lib and I have few sponsors. My lib also depends on another lib that has a sponsorship page. Adding the ability to forward the sponsorship will be a great encouragement to sponsor others.

Looking forward to this awesome feature 👀

**Comment by alvarosabu** [comment]
+100000 To this!!

**Comment by bytrangle** [comment]
This would also help third-party contributors greatly. Maintainers always have more requests than they can handle. They use their free time to contribute to open source projects, yet they will never get enough visibility. And because they don't own the projects they work on, it's hard to call for sponsorship on their own.
